CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Added
+- `propagate_flags` keyword for `UVData.frequency_average` for different resulting flags from averaging
```

Comment:
Can you make this log message a little clearer in terms of what the new functionality is?